### PR TITLE
Fix big question mark

### PR DESCRIPTION
**Pull request by AlexeyKovalkovvv**

### CODE REVIEW

File: splitit-installment-payments-enabler/assets/css/splitit-front.css [hunk 1]
```diff
@@ -43,4 +43,6 @@ span.splitit-installment-price-checkout img.logoWidthSrc{/* width:44px !importan
 }
 .related .splitit-installment-price-product, .related #tell-me-more {display: none !important;}
 
-div.payment_box.payment_method_splitit img.tell-me-more-image {max-height: 100% !important;}
\ No newline at end of file
+div.payment_box.payment_method_splitit img.tell-me-more-image {max-height: 100% !important;}
```
Comment (sixg) [inline splitit-installment-payments-enabler/assets/css/splitit-front.css:46]: @AlexeyKovalkovvv has this been tested ?

**Review by sixg — CHANGES_REQUESTED** [review]
@AlexeyKovalkovvv Please review the changes and remove conflicts before the PR can be merged.

File: splitit-installment-payments-enabler/assets/css/splitit-front.css (hunk 1 above)
Comment (AlexeyKovalkovvv) [reply splitit-installment-payments-enabler/assets/css/splitit-front.css]: Removed locally, it works fine without this line

Comment (sixg) [reply splitit-installment-payments-enabler/assets/css/splitit-front.css]: ok, thanks for confirmation.

**Review by sixg — APPROVED** [review]
@AlexeyKovalkovvv tested it properly and confirmed that the PR can now be merged. The changes seems to be fine. Staging needs to be tested again after the merge.
PR approved.

**Pull request merged**